python/ray/data/_internal/compute.py

```diff
@@ -295,7 +297,8 @@ def map_block_nosplit(
                 if not ready:
                     if (
                         len(workers) < self.max_size
-                        and len(ready_workers) / len(workers) > 0.8
+                        and len(ready_workers) / len(workers)
+                        > self.ready_to_total_workers_ratio
```

Comment:
This looks weird format to me, but this is the result after running `scripts/format.sh`.